content/docs/next-release/deployment.md

```diff
@@ -306,6 +306,25 @@ Rollover index management strategy is more complex than using the default daily
 To learn more about rollover index management in Jaeger refer to this
 [article](https://medium.com/jaegertracing/using-elasticsearch-rollover-to-manage-indices-8b3d0c77915d).
 
+#### Elasticsearch ILM support
+{{< warning >}}
+Experimental Feature, Added in release 1.22.
+
+Supported versions: 7.x
+{{< /warning >}}
+[Elasticsearch ILM](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html) helps us automatically manage indices according to our performance, resiliency, and retention requirements.
+For example, you could use ILM to:
+
+* Spin up a new index when an index reaches a certain size or number of documents
+* Create a new index each day, week, or month and archive previous ones
+* Delete stale indices to enforce data retention standards
+
+ILM usage for jaeger indices can be enabled by following below steps:
+
+- Create an ILM policy in elasticsearch named *jaeger-ilm-policy*
+- Run elasticsearch initializer with ES_USE_ILM=true in environment variables. (details in below section)
+- After the initialization, deploy Jaeger with `--es.use-ilm=true` and `--es.use-aliases=true`.
```

Comment:
I propose we replace this block with (and remove the changes from L336 onwards):

##### Enabling ILM support

1. Create an ILM policy in elasticsearch named *jaeger-ilm-policy*.

   For example, the following policy will rollover the "active" index when it is
   older than 1m and delete indices that are older than 2m.
   ```
   curl -X PUT http://localhost:9200/_ilm/policy/jaeger-ilm-policy \
   -H 'Content-Type: application/json; charset=utf-8' \
   --data-binary @- << EOF
   {
     "policy": {
       "phases": {
         "hot": {
           "min_age": "0ms",
           "actions": {
             "rollover": {
               "max_age": "1m"
             },
             "set_priority": {
               "priority": 100
             }
           }
         },
         "delete": {
           "min_age": "2m",
           "actions": {
             "delete": {}
           }
         }
       }
     }
   }
   EOF
   ```
2. Run elasticsearch initializer with `ES_USE_ILM=true`:
   ```
   docker run -it --rm --net=host -e ES_USE_ILM=true jaegertracing/jaeger-es-rollover:latest init http://localhost:9200 # <1>
   ```
   <1> If you need to initialize archive storage, add `-e ARCHIVE=true`.

   {{< info >}}
   While initializing with ILM support, make sure that an ILM policy named *jaeger-ilm-policy* is created in Elasticsearch beforehand.
   Initialization checks for this policy - if it does not exist, it would error out with following message.

   *ILM policy jaeger-ilm-policy doesn't exist in Elasticsearch. Please create it and rerun init*
   {{< /info >}}
3. After the initialization, deploy Jaeger with `--es.use-ilm=true` and `--es.use-aliases=true`.

Reply:
Agreed, adding it. Just wanted mention something with respect to above ILM policy to be accurate, even though the max age in above policy is set to 1m,  rollover would only happen after 10m - as indices.lifecycle.poll_interval is set to 10 mins by default. 
This can be changed in cluster settings though. :)